addons/binding/org.openhab.binding.mcp23017/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="mcp23017" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="mcp23017switch">
```

Comment:
Ok, 
so should I do 16 channels (A0...A7, B0...B7) of type "inputPin" and 16 channels of type "outputPin"? And how to support default state of PIN (LOW, HIGH)? As a parameter of item?